.github/BOTMETA.yml

```diff
@@ -206,6 +206,7 @@ files:
     maintainers: chouseknecht maxamillion fabianvf flaper87 willthames
     labels: k8s
   $modules/clustering/k8s/_kubernetes.py: supertom
+  $modules/clustering/etcd3.py: $team_suse
```

Comment:
This probably should not be `$team_suse`. It's fine if you would like to individually added as a maintainer for `etcd3`, just add your name.

Reply:
@evrardjp please follow sam's advice and change this entry.

Reply:
Will do.